XenModel/Alerts/PerfmonOptionsDefinition.cs

```diff
@@ -117,11 +197,12 @@ public static PerfmonOptionsDefinition GetPerfmonOptionsDefinitions(IXenObject x
 
             string mailDestination = GetMailDestination(connection);
             string mailHub = GetSmtpMailHub(connection);
+            string mailLanguageCode = GetMailLanguageCode(connection);
 
             if (mailDestination == null || mailDestination == null)

```

Comment:
Unrelated to this commit, I think the second condition in this `if` was supposed to be `mailHub == null`